vendor/sigs.k8s.io/cluster-api/config/BUILD

```diff
@@ -0,0 +1,10 @@
+filegroup(
+  name = "kustomize-yaml",
+  srcs = glob([
+        "crds/*.yaml",
+        "rbac/*.yaml",
+        "manager/*.yaml",
+        "default/*.yaml"
+  ]),
+  visibility = ["//visibility:public"],
+)
```

Comment:
I don't think this file will be preserved through a `dep ensure -update`, is there a way to host this config outside of vendor?

Reply:
Likely, since `dep` deletes the old vendor directory. Would be fine if it was just Gazelle.